nbs/index.ipynb

```diff
@@ -1,23 +1,52 @@
 {
```

Comment:
<p>so just add networkx to the dependencies, no need to ask the user to install it. They do need to import it to create their graphs though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>